packages/odyssey/src/scss/abstracts/_functions.scss

```diff
@@ -72,13 +72,14 @@
 
 $icons: (
   'arrow':'<svg viewBox="0 0 10 12" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M4.29289 0.292893C4.68342 -0.0976311 5.31658 -0.0976311 5.70711 0.292893L9.70711 4.29289C10.0976 4.68342 10.0976 5.31658 9.70711 5.70711C9.31658 6.09763 8.68342 6.09763 8.29289 5.70711L6 3.41421V11C6 11.5523 5.55228 12 5 12C4.44772 12 4 11.5523 4 11V3.41421L1.70711 5.70711C1.31658 6.09763 0.683417 6.09763 0.292893 5.70711C-0.0976311 5.31658 -0.0976311 4.68342 0.292893 4.29289L4.29289 0.292893Z" fill="%%COLOR%%"/></svg>',
+  'anchor': '<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M14.2644 9.73252C14.0207 9.4874 13.7505 9.27011 13.4588 9.08464C12.6863 8.58993 11.7881 8.32705 10.8707 8.32705C9.5974 8.32468 8.37583 8.8306 7.47704 9.73252L3.11668 14.0963C2.21941 14.9953 1.7151 16.2133 1.7143 17.4835C1.71261 20.134 3.85985 22.284 6.51035 22.2857C7.7817 22.2901 9.00214 21.7866 9.90062 20.887L13.5 17.2877C13.5654 17.2229 13.6019 17.1344 13.6014 17.0423C13.6003 16.853 13.4459 16.7004 13.2566 16.7015H13.1195C12.3675 16.7042 11.6222 16.5598 10.9256 16.2765C10.7972 16.2236 10.6497 16.2535 10.552 16.3519L7.96383 18.9434C7.16017 19.7471 5.85717 19.7471 5.0535 18.9434C4.24983 18.1397 4.24983 16.8367 5.0535 16.0331L9.43101 11.659C10.234 10.857 11.5349 10.857 12.3379 11.659C12.8791 12.1684 13.7233 12.1684 14.2644 11.659C14.4973 11.426 14.6384 11.1169 14.6621 10.7883C14.6871 10.3956 14.5423 10.0111 14.2644 9.73252Z" fill="%%COLOR%%"/><path d="M20.8769 3.11999C19.0026 1.24571 15.9638 1.24571 14.0895 3.11999L10.4936 6.71247C10.3949 6.81169 10.3664 6.96093 10.4216 7.08956C10.4762 7.21855 10.6038 7.30134 10.7438 7.29865H10.8707C11.6218 7.29729 12.3659 7.44283 13.0611 7.72716C13.1895 7.77998 13.337 7.75018 13.4348 7.65176L16.016 5.07392C16.8197 4.27025 18.1227 4.27025 18.9263 5.07392C19.73 5.87759 19.73 7.18059 18.9263 7.98426L15.7109 11.1962L15.6835 11.2271L14.5591 12.3446C13.7561 13.1466 12.4552 13.1466 11.6522 12.3446C11.111 11.8353 10.2668 11.8353 9.7257 12.3446C9.49138 12.5794 9.3501 12.8912 9.32805 13.2222C9.30298 13.6148 9.4478 13.9994 9.7257 14.278C10.1225 14.6764 10.5867 15.0014 11.0969 15.2378C11.1689 15.2721 11.2408 15.2995 11.3128 15.3304C11.3848 15.3612 11.4602 15.3852 11.5322 15.4126C11.6042 15.4401 11.6796 15.464 11.7516 15.4846L11.9538 15.5394C12.0909 15.5737 12.2281 15.6011 12.3686 15.6251C12.5379 15.6503 12.7084 15.6663 12.8794 15.6731H13.1193H13.1399L13.3456 15.6492C13.421 15.6457 13.4998 15.6286 13.589 15.6286H13.7055L13.942 15.5943L14.0517 15.5738L14.2506 15.5326H14.2883C15.1303 15.3211 15.8992 14.8851 16.513 14.2711L20.8768 9.90734C22.7512 8.03306 22.7512 4.99426 20.8769 3.11999Z" fill="%%COLOR%%"/></svg>',
```

Comment:
This should get run through SVGO to strip the width/height and anything else non-standard.

Reply:
Fixed